frontend/src/features/admin-form/settings/mutations.ts

```diff
@@ -300,6 +301,25 @@ export const useMutateFormSettings = () => {
     },
   )
 
+  const mutateMrfStatusTracker = useMutation(
+    (nextHasStatusTracker: boolean) =>
+      updateMrfStatusTracker(formId, {
+        hasStatusTracker: nextHasStatusTracker,
+      }),
+    {
+      onSuccess: (newData) => {
+        const hasStatusTracker =
+          'hasStatusTracker' in newData ? newData.hasStatusTracker : false
+
+        handleSuccess({
+          newData,
+          toastDescription: `${t('features.adminForm.toasts.statusTracker.successBefore')} ${hasStatusTracker ? '' : t('features.adminForm.toasts.statusTracker.disabled')}${t('features.adminForm.toasts.statusTracker.successAfter')}`,
```

Comment:
Somewhat difficult for us to read the translations 🤔. Thinking that we might be able to look into building a vscode extension to ease this in the future.

Reply:
sorry for hijacking the convo (am awake and bored :p)
- extension: https://github.com/lokalise/i18n-ally
- deep nesting can be tackled with: [keyPrefix](https://react.i18next.com/latest/usetranslation-hook#optional-keyprefix-option)